toolz/itertoolz.py

```diff
@@ -495,6 +497,22 @@ def concatv(*seqs):
     return concat(seqs)
 
 
+def repeat_each(n, seq):
+    """ Repeats each element multiple times.
+
+    >>> list(repeat_each(2, [1, 2, 3]))
+    [1, 1, 2, 2, 3, 3]
+    """
+
+    if n < 0:
+        raise ValueError('n must be positive, but got %r' % n)
+
+    if (n % 1) != 0:
```

Comment:
Interesting. Apparently Python 2.6 allows for non-integer arguments. Should we add it back?

```python
>>> list(repeat_each(0.5, [1, 2, 3]))
toolz/functoolz.py:637: DeprecationWarning: integer argument expected, got float
  return func(b, a)
```